src/api/entities/Identity/index.ts

```diff
@@ -524,7 +522,7 @@ export class Identity extends Entity<UniqueIdentifiers, string> {
       return null;
     }
 
-    return scopeIdToString(scopeId);
+    return identityIdToString(scopeId);
```

Comment:
Why ScopeId is now being generated everywhere as an IdentityId?

Reply:
Under the hood both are the same and new lookup types generated point them as IdentityId